vite.config.js

```diff
@@ -26,6 +26,7 @@ export default defineConfig({
         defaultHandler(warning);
       },
     },
+    cssMinify: false,
```

Comment:
What's the issue here? we say temporarily, but we do not describe the issue here, nor some relevant discussion where we can assume that a fix will appear.